src/zc/table/fieldcolumn.py

```diff
@@ -38,9 +40,10 @@ def toSafe(string):
         string = ''.join(string.encode('base64').split())
     return string
 
+
 class BaseColumn(column.Column):
 
-    ###### subclass helper API (not expected to be overridden) ######
+    # subclass helper API (not expected to be overridden) ######
```

Comment:
If you're removing the `#####` decoration from the front of the comment, remove it from the back as we..